module/zfs/dbuf.c

```diff
@@ -272,13 +337,15 @@ dbuf_hash_insert(dmu_buf_impl_t *db)
 	int level = db->db_level;
 	uint64_t blkid, hv, idx;
 	dmu_buf_impl_t *dbf;
+	uint64_t depth = 0;
 
 	blkid = db->db_blkid;
 	hv = dbuf_hash(os, obj, level, blkid);
 	idx = hv & h->hash_table_mask;
 
 	mutex_enter(DBUF_HASH_MUTEX(h, idx));
 	for (dbf = h->hash_table[idx]; dbf != NULL; dbf = dbf->db_hash_next) {
+		depth++;
```

Comment:
It'd be nice to model this for loop after the one in `buf_hash_insert()`.  It's fine to use `i` for the variable name here instead of `depth`.